### ISSUE TITLE
Issue with installing any Unity scene to my iOS device

### ISSUE THREAD

**Issue by ansyal2006** [asus4/tf-lite-unity-sample#30]
The scenes run fine in Unity but the application does not get installed in my iOS device (iphone8) through XCode. I have checked ARM64. But donot know how to eliminate the error. Can somebody please help?
<img width="839" alt="Screenshot 2020-06-30 at 4 51 01 PM" src="https://user-images.githubusercontent.com/22514740/86120436-ecbe4a80-baf1-11ea-8a6b-c23e2c82dfe1.png">

**Comment by ewokdanceparty** [comment]
I had the same problem, I believe. To fix it, I followed the steps under "How to build tensorflow lite for Unity" in the instructions, with the following caveats:

-I had to install bazel with `brew install bazel` which gave me bazel 3.3.0. Tensorflow wanted 3.1.0, so I changed that to 3.3.0 in the .bazelversion file in the cloned Tensorflow repo.

-Something initially went wrong with step 3, I can't remember what, but the following worked: `./build_tflite.py --tfpath ../tensorflow -ios` I think I declined all of the options in the Tensorflow install except for iOS support.
